fbw-common/src/systems/instruments/src/EFB/Settings/ThrottleConfig/BaseThrottleConfig.tsx

```diff
@@ -53,7 +53,14 @@ export const BaseThrottleConfig: FC<BaseThrottleConfigProps> = ({
   activeDetent,
   reverseDisabled,
 }) => {
-  const [throttlePosition] = useSimVar(`L:A32NX_THROTTLE_MAPPING_INPUT:${axisNumber}`, 'number', 30);
+  let inputAxisNumber = axisNumber;
+  // this is required the users hardware axis 2 is mapped to logical input axis 3 in the sim
+  // A380 case (4 throttles) with 2 user hardware axes and the second user hardware axis is the current axis to be displayed
+  if (numberOfThrottles === 4 && numberOfAxis === 2 && axisNumber === 2) {
+    inputAxisNumber = 3;
+  }
+  const [throttlePosition] = useSimVar(`L:A32NX_THROTTLE_MAPPING_INPUT:${inputAxisNumber}`, 'number', 30);
+
```

Comment:
Prefer to add a new param to control the inputAxisNumber rather than do this. As discussed.

Reply:
i.e.
```
/**
 * BaseThrottleConfigProps are the props for the BaseThrottleConfig component.
 * @param className             css class pass through
 * @param axisNumber            number of the current MSFS bind axis
 * @param inputAxis                  Input throttle axis
 * @param numberOfAxis          number of axis that are mapped
 * @param numberOfThrottles     number of throttles that are mapped
 * @param throttleSimvarsSet1   array of throttle simvars to map the axis values to
 * @param throttleSimvarsSet2   array of throttle simvars to map the axis values to
 * @param throttleSimvarsSet3   array of throttle simvars to map the axis values to
 * @param throttleSimvarsSet4   array of throttle simvars to map the axis values to
 * @param activeDetent          currently active detent
 * @param reverseDisabled       boolean to disable reverse detent for the axis
 */
interface BaseThrottleConfigProps {
  className?: string;
  axisNumber: number;
  inputAxis: number;
  numberOfAxis: number;
  numberOfThrottles: number;
  throttleSimvarsSet1?: ThrottleSimvar[];
  throttleSimvarsSet2?: ThrottleSimvar[];
  throttleSimvarsSet3?: ThrottleSimvar[];
  throttleSimvarsSet4?: ThrottleSimvar[];
  activeDetent: number;
  reverseDisabled?: boolean;
}
```